sig-network/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
```

Comment:
This is my suggestion to this section:

> SIG Network is responsible for all network capabilities in Kubevirt, with the high level goal of enabling Virtual Machine to connect to networks in the Kubernetes ecosystem.
> The SIG Network owns the API,  implementation and documentation related to all network functionalities.

Reply:
I liked better the previous option. Let's see what other reviewers think.

Reply:
both are fine imho, but i am in favor of the current, it is bit more specific

Reply:
IMO it is not a fit to make this specific, it is a charter which should not change when we add or remove features.

Said that, I will not insist further.

Reply:
I like this suggestion, we can elaborate with more details at `Code, Binaries and Services`.